schemas/mldsa_verify_schema.json

```diff
@@ -7,11 +7,16 @@
         "type": {
           "enum": ["MlDsaVerify"]
         },
-        "public_key": {
+        "publicKey": {
```

Comment:
> Also fix a typo in mldsa_verify_schema.json. This went unnoticed because both missing and extra fields are allowed.

Argh :face_exhaling: Thanks for fixing. Such a frustrating default for a schema language!

I thought we managed to get `"additionalProperties": false` in all the schemas but this one must have been overlooked. A quick `grep -rL '"additionalProperties"' schemas/` on `main` only flagged `mldsa_verify_schema.json`.

Would you be interested in sprinkling `additionalProperties` & some `"required"` fields into this schema's definitions as part of this PR, or would you prefer I do it as a follow-up?